metaseq/data/data_utils.py

```diff
@@ -375,154 +273,6 @@ def post_process(sentence: str, symbol: str):
     return sentence
 
 
-def compute_mask_indices(
```

Comment:
Can you double check armen isn’t using this in his branch? It looks like what would normally power bart and therefore cm3